src/sentry/api/endpoints/source_map_debug.py

```diff
@@ -191,9 +176,122 @@ def _find_matches(self, release_artifacts, unified_path):
             artifact
             for artifact in release_artifacts
             if artifact.name.endswith(unified_path.split("/")[-1])
+            if artifact.name.endswith(unified_path.split("/")[-1])
         ]
         return full_matches, partial_matches
 
+    def _extract_release(self, event, project):
+        release_version = event.get_tag("sentry:release")
+
+        if not release_version:
+            raise Exception
+        try:
+            release = Release.objects.get(
+                organization=project.organization, version=release_version
+            )
+        except Release.DoesNotExist:
+            raise Exception
+        return release
```

Comment:
Given that you're catching `Exception` you could code-golf this down

```suggestion
        return Release.objects.get(
            organization=project.organization, version=release_version
        )
```

If the release query raises catching `Exception` will do the right thing.